update readme and contributing (#1)
quizlet/hammock: README.md

# Warning

Direct usage of internal classes (especially the non-disposable ones) is strongly discouraged.

# Contributing
If you ever wanted to contribute to open source, now is the chance!

> Please read [CONTRIBUTING.md](https://github.com/quizlet/hammock/blob/master/CONTRIBUTING.md) for details and the process for submitting pull requests to us.

Reviewer: ```suggestion
Please read [CONTRIBUTING.md](https://github.com/quizlet/hammock/blob/master/CONTRIBUTING.md) for details and the process for submitting pull requests.
```

# Versioning
We use GitHub for versioning. For the versions available, see the [tags on this repository](https://github.com/quizlet/hammock/tags).

# Contributors
Thanks to the following people who have contributed to this project:
- [Tyron Jung](https://github.com/tyronjung-quizlet)
- [Riya Dashoriya](https://github.com/riyadashoriya-qz)
- [Karoun Kasraie](https://github.com/karoun)
- [Andrew Sutherland](https://github.com/asuth)
- [Josh Rai](https://github.com/joshrai)
- [Shaobo Sun](https://github.com/shaobos)

# Acknowledgements
- [Turadg Aleahmad](https://github.com/turadg)

> - Sean Young

Reviewer: I think Sean also has a Quizlet GitHub.

# Contact

> Please reach out to tyron.jung@quizlet.com and riya.dashoriya@quizlet.com with any questions.

Reviewer: ```suggestion
Please reach out to tyron.jung@quizlet.com or riya.dashoriya@quizlet.com if you have any questions.
```

# License

> [MIT](https://github.com/quizlet/hammock/blob/master/README.md)

Reviewer: Should this link to the `LICENSE` file?
